### PR TITLE
add documentation for submit job no wait and submit job and wait temp…

### PR DESCRIPTION
**Pull request by sophiesongge**
…lates

### CODE REVIEW

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -1532,6 +1532,16 @@ Notifications will only be sent for jobs whose generic information contains user
 </genericInformation>
 ----
 
+=== Chaining Jobs
+
+The SubmitJobNoWait and SubmitJobAndWait templates under the *Controls* menu allow you to submit a workflow from another workflow through using *schedulerapi*.
```
Comment (marcocast) [inline src/docs/user/ProActiveUserGuide.adoc]: from another workflow using the *schedulerapi*.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -1532,6 +1532,16 @@ Notifications will only be sent for jobs whose generic information contains user
...
+
+In order to submit a workflow, you need to put the URL of that workflow in Task Variables under the key *workflowURL*. You can use whatever URL, if that URL is
```
Comment (marcocast) [inline src/docs/user/ProActiveUserGuide.adoc]: .. that workflow in the Task Variables.....
...You can use any URL that points to a valid workflow xml file.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 3]
```diff
@@ -1532,6 +1532,16 @@ Notifications will only be sent for jobs whose generic information contains user
...
+a valid workflow xml file. For example, you can add the workflow that you want to execute into *Workflow Catalog*, then retrieve its URL by indicating its Bucket ID
+and workflow ID. Moreover, the SubmitJobAndWait template will verify the status of the workflow it submits every minutes until that workflow is finished, while SubmitJobNoWait
```
Comment (marcocast) [inline src/docs/user/ProActiveUserGuide.adoc]: The SubmitJobAndWait will wait for the completion of the submitted workflow before proceeding further with the execution of the dependant tasks.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 4]
```diff
@@ -1532,6 +1532,13 @@ Notifications will only be sent for jobs whose generic information contains user
 </genericInformation>
 ----
 
+=== Chaining Jobs
+
+The SubmitJobNoWait and SubmitJobAndWait templates under the *Controls* menu allow you to submit a workflow from another workflow using *schedulerapi*.
```
Comment (mbenguig) [inline src/docs/user/ProActiveUserGuide.adoc]: remove the

File: src/docs/user/ProActiveUserGuide.adoc [hunk 5]
```diff
@@ -1532,6 +1532,13 @@ Notifications will only be sent for jobs whose generic information contains user
...
+
+In order to submit a workflow, you need to put the URL of that workflow in the Task Variables under the key *workflowURL*. You can use any URL that points to
+a valid workflow xml file. For example, you can add the workflow that you want to execute into *Workflow Catalog*, then retrieve its URL by indicating its Bucket ID
+and workflow ID. Moreover, the SubmitJobAndWait will wait for the completion of the submitted workflow before proceeding further with the execution of the dependant tasks.
```
Comment (mbenguig) [inline src/docs/user/ProActiveUserGuide.adoc]: remove the

**Comment by mbenguig** [comment]
ok (2 comments)

**Pull request merged**